cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -74,10 +64,8 @@ function create-resource-from-string() {
 # managed result is of that. Start everything below that directory.
 echo "== Kubernetes addon manager started at $(date -Is) with ADDON_CHECK_INTERVAL_SEC=${ADDON_CHECK_INTERVAL_SEC} =="
 
-ensure_python
-
 # Create the namespace that will be used to host the cluster-level add-ons.
-start_addon /etc/kubernetes/addons/namespace.yaml 100 10 "" &
+start_addon /opt/namespace.yaml 100 10 "" &
```

Comment:
Sorry for late comment. Is this the path of the instance, or the path inside a container? If it is the former, this will not work on GCI.